lib/frontmatter.js

```diff
@@ -100,6 +100,11 @@ const schema = {
     },
     interactive: {
       type: 'boolean'
+    },
+    // Platform-specific content preference
+    defaultPlatform: {
+      type: 'string',
+      enum: ['mac', 'windows', 'linux'] // TODO: add 'all'?
```

Comment:
How would 'all' work?

Reply:
I have that question too. I'm not sure how it is intended to work, it's also not used anywhere, but described here: https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#operating-system-tags

Reply:
I think we can omit it because:

* The equivalent could be to just not include `defaultPlatform` in the frontmatter, and existing behavior will prevail
* We don't... actually... use `{% all %}` anywhere in the codebase 🙃

Reply:
Yeah, I think it can be left out. The tag is a bit of a misnomer, it doesn't mean "all platforms" but rather "all other platforms" or "another unspecified platform". If something actually applies to all platforms, then you would simply not wrap that content in any of the platform tags. I don't see how `{% all %}` is useful to be honest and would even go as far as to remove this tag/block entirely.